charts/anchor-platform/sep-service/templates/configmap.yaml

```diff
@@ -1,7 +1,19 @@
+{{- if .Values.assets }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ .Values.fullName }}-assets
+data:
+  assets.json: |
+    {
+      "assets": {{ toPrettyJson .Values.assets | indent 4 }}
+    }
```

Comment:
This looks fine but I'm wondering what's easier for end users.
Maybe making the value itself be just embedded JSON string would make it easier?
That would cause us to loose some data validation because to yaml it would be just a string. But at the same time it might be helpful to allow straight copy/paste from examples or docs in some other places?

Not a strong opinion though

Reply:
I think the correct format of this file is somewhat debated, and it's moved again in the configuration refactor, which will follow this